### PR TITLE
Fix #16888 - JS error on renaming tables in saved Designer page

### LINKED ISSUE #16888: Cannot read property 'style' of null
### Describe the bug

```json
{
    "pma_version": "5.1.1-dev",
    "browser_name": "CHROME",
    "browser_version": "90.0.4430.93",
    "user_os": "Linux",
    "server_software": "nginx/1.18.0",
    "user_agent_string": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.93 Safari/537.36",
    "locale": "fr",
    "configuration_storage": "disabled",
    "php_version": "7.4.15",
    "script_name": "index.php",
    "exception_type": "js",
    "exception": {
        "mode": "stack",
        "name": "TypeError",
        "message": "Cannot read property 'style' of null",
        "stack": [
            {
                "func": "?",
                "line": "164",
                "column": "13",
                "context": [
                    "    DesignerMove.markSaved();",
                    "",
                    "    for (var t = 0; t < tblCords.length; t++) {",
                    "      var tbId = db + '.' + tblCords[t].tableName;",
                    "      var table = document.getElementById(tbId);",
                    "      table.style.top = tblCords[t].y + 'px';",
                    "      table.style.left = tblCords[t].x + 'px';",
                    "      var checkbox = document.getElementById('check_vis_' + tbId);",
                    "      checkbox.checked = true;",
                    "      DesignerMove.visibleTab(checkbox, checkbox.value);",
                    "    }"
                ],
                "uri": "js/dist/designer/page.js?v=5.1.1-dev",
                "scriptname": "js/dist/designer/page.js"
            },
            {
                "func": "?",
                "line": "186",
                "column": "13",
                "context": [
                    "      DesignerOfflineDB.loadObject('table_coords', page.tblCords[i], functi//...",
                    "        tblCords.push(tblCord);",
                    "",
                    "        if (tblCords.length === count) {",
                    "          if (typeof callback !== 'undefined') {",
                    "            callback(page, tblCords);",
                    "          }",
                    "        }",
                    "      });",
                    "    }",
                    "  });"
                ],
                "uri": "js/dist/designer/page.js?v=5.1.1-dev",
                "scriptname": "js/dist/designer/page.js"
            },
            {
                "func": "IDBRequest.cursorRequest.onsuccess",
                "line": "58",
                "column": "7",
                "context": [
                    "    var transaction = db.transaction([table], 'readwrite');",
                    "    var objStore = transaction.objectStore(table);",
                    "    var cursorRequest = objStore.get(parseInt(id));",
                    "",
                    "    cursorRequest.onsuccess = function (e) {",
                    "      callback(e.target.result);",
                    "    };",
                    "",
                    "    cursorRequest.onerror = designerDB.onerror;",
                    "  };",
                    ""
                ],
                "uri": "js/dist/designer/database.js?v=5.1.1-dev",
                "scriptname": "js/dist/designer/database.js"
            }
        ],
        "uri": "index.php?route=%2Fdatabase%2Fdesigner"
    }
}
```

### To Reproduce

1. Go to Designer

### Expected behavior

No error

### Server configuration

- phpMyAdmin version: 5.1

### PR DESCRIPTION
**Pull request by shucon**
Signed-off-by: Saksham Gupta <shucon01@gmail.com>

### Description

Please describe your pull request.

Fixes #16888 

Before submitting pull request, please review the following checklist:

- [ ] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [ ] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [ ] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [ ] Every commit has a descriptive commit message.
- [ ] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [ ] Any new functionality is covered by tests.

### CODE REVIEW

File: js/src/designer/page.js [hunk 1]
```diff
@@ -128,20 +128,29 @@ DesignerPage.showNewPageTables = function (check) {
 };
 
 DesignerPage.loadHtmlForPage = function (pageId) {
-    DesignerPage.showNewPageTables(false);
+    DesignerPage.showNewPageTables(true);
     DesignerPage.loadPageObjects(pageId, function (page, tblCords) {
         $('#name-panel').find('#page_name').text(page.pageDescr);
-        DesignerMove.markSaved();
+        var tableMissing = 0;
```
Comment (williamdes) [inline js/src/designer/page.js:134]:
```suggestion
        var tableMissing = false;
```

File: js/src/designer/page.js [hunk 2]
```diff
@@ -128,20 +128,29 @@ DesignerPage.showNewPageTables = function (check) {
...
         for (var t = 0; t < tblCords.length; t++) {
             var tbId = db + '.' + tblCords[t].tableName;
             var table = document.getElementById(tbId);
+            if (table === null) {
+                tableMissing = 1;
```
Comment (williamdes) [inline js/src/designer/page.js:139]:
```suggestion
                tableMissing = true;
```

File: js/src/designer/page.js [hunk 3]
```diff
@@ -128,20 +128,29 @@ DesignerPage.showNewPageTables = function (check) {
...
+            }
             table.style.top = tblCords[t].y + 'px';
             table.style.left = tblCords[t].x + 'px';
 
             var checkbox = document.getElementById('check_vis_' + tbId);
             checkbox.checked = true;
             DesignerMove.visibleTab(checkbox, checkbox.value);
         }
+        DesignerMove.markSaved();
+        if (tableMissing === 1) {
```
Comment (williamdes) [inline js/src/designer/page.js:150]:
```suggestion
        if (tableMissing === true) {
```

File: libraries/classes/Controllers/JavaScriptMessagesController.php [hunk 4]
```diff
@@ -538,6 +538,7 @@ private function setMessages(): void
             'strSavePageAs' => __('Save page as'),
             'strOpenPage' => __('Open page'),
             'strDeletePage' => __('Delete page'),
+            'strSavedPageTableMissing' => __('Some tables saved in this page might be renamed or removed.'),
```
Comment (williamdes) [inline libraries/classes/Controllers/JavaScriptMessagesController.php:541]:
```suggestion
            'strSavedPageTableMissing' => __('Some tables saved in this page might have been renamed or deleted.'),
```

File: js/src/designer/page.js [hunk 5]
```diff
@@ -128,20 +128,29 @@ DesignerPage.showNewPageTables = function (check) {
 };
 
 DesignerPage.loadHtmlForPage = function (pageId) {
-    DesignerPage.showNewPageTables(false);
+    DesignerPage.showNewPageTables(true);
```
Comment (williamdes) [inline js/src/designer/page.js:131]: Could you explain this change please?

**Review by williamdes — APPROVED** [review]
Looks good to me

File: js/src/designer/page.js (hunk 5 above)
Comment (shucon) [reply js/src/designer/page.js:131]: It shows any tables that were renamed or added in the database after page save. Else, all such tables would be hidden.

**Review by MauricioFauth — APPROVED** [review]

**Pull request merged**